enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/docverify/zenid/provider/ZenidDocumentVerificationProvider.java

```diff
@@ -497,13 +483,13 @@ private DocumentsVerificationResult toResult(OwnerId id, ZenidWebInvestigateResp
             List<DocumentVerificationResult> verificationResults = new ArrayList<>();
 
             String extractedData = toExtractedData(id, response.getMinedData());
-            for (String sampleId : sampleIdsValidations.keySet()) {
+            for (var entry : sampleIdsValidations.entrySet()) {
```

Comment:
Is there any benefit to using var? In my opinion this usage hides the type and makes code less portable, so unless we plan to use var everywhere, I would not use it.

Reply:
We are changing `keySet` to `entrySet`, so it should be `Map.Entry<String, List<ZenidWebInvestigationValidatorResponse>>` which is less readable than `var`.

Reply:
Ok.